compair/static/modules/assignment/assignment-module.js

```diff
@@ -1402,7 +1402,8 @@ module.controller("AssignmentWriteController",
                 rank_display_limit: null,
                 answer_grade_weight: 1,
                 comparison_grade_weight: 1,
-                self_evaluation_grade_weight: 1
+                self_evaluation_grade_weight: 1,
+                existingFile: false
```

Comment:
The CI tests are failing.  Need to add this new property (`existingFile: false`) to the test cases too

https://github.com/ubc/compair/blob/05ef114f7038a6e95c069ace46d859e4c9ba7314/compair/static/modules/assignment/assignment-module_spec.js#L1025-L1035

Reply:
Thank you, can never remember to update the tests...